src/css/controls/imports/icons.less

```diff
@@ -7,6 +7,10 @@
 .jw-icon {
     .square(44px);
     background-color: transparent;
+
+    &:not(.jw-no-focus)&:focus svg {
```

Comment:
this outputs to `.jw-icon:not(.jw-no-focus).jw-icon:focus svg`; when do we have this ?